Solutions/SIGNL4/Data Connectors/Logo/signl4.svg

```diff
@@ -0,0 +1,224 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
```

Comment:
Please remove xmlns link. Please refer [guidelines](https://microsoft.sharepoint.com/:x:/r/teams/AzureSecurityInsights/_layouts/15/guestaccess.aspx?e=aETsaQ&share=EWs_HuWia89Ns0pc1jXRWQkB7aNXX6Zi1REQcKtH4gQp5Q&isSPOFile=1).

Reply:
OK and done. All xmlns linke removed.